### ISSUE TITLE
RFC-3: Reserve versions that are ambiguous with JSON

### ISSUE THREAD

**Issue by justmoon** [interledger/rfcs#27]
The curly brace `{` corresponds to byte `0x7B`. Perhaps we should reserve interledger versions `0x7B00` - `0x7BFF` => 31488-31743 should be reserved to ensure the binary format is always distinguishable from the JSON encoded format.

**Comment by emschwartz** [comment]
Is this still needed?

**Comment by sharafian** [comment]
currently, the routing protocol uses stringified JSON in the `ilp` protocolData of BTP. Seems worth keeping around for now, unless we decide that the routing protocol should be switched to a 0-amount ILP prepare with a `peer.` address

**Comment by justmoon** [comment]
Yup, this is currently being used in the connector implementation for CCPv1.

CCPv2 should use something else obv. Still think we should reserve that version byte permanently, it's just nice in general to be able to distinguish and ILP packet from some JSON blob that somehow ended up in binary.
